runtime/src/migration.rs

```diff
@@ -47,8 +37,8 @@ decl_storage! {
         /// initialized to Some(VERSION.spec_version). It is an Option because the first time the module
         /// was introduced was as a runtime upgrade and type was never changed.
         /// When the runtime is upgraded the spec version be updated.
-        pub SpecVersion get(spec_version) build(|_config: &GenesisConfig| {
-            VERSION.spec_version
+        pub SpecVersion get(fn spec_version) build(|_config: &GenesisConfig| {
```

Comment:
There is a fundamental problem with this migration module, specifically to do with what the initial value set here will be. This has actually only caused a problem with unexpected behavior when working on our staging networks when. I won't go into the details, but we should as previously discussed move away from it to use the new builtin substrate on runtime upgraded callback.

Reply:
Ok. I will remove the migration module out of the runtime.